src/metabase/driver/firebolt.clj

```diff
@@ -333,28 +342,24 @@
 
 ;-------------------------Supported features---------------------------
 
-(defmethod driver/database-supports? [:firebolt :basic-aggregations]  [_ _] true)
-
-(defmethod driver/database-supports? [:firebolt :expression-aggregations]  [_ _] true)
-
-(defmethod driver/database-supports? [:firebolt :percentile-aggregations]  [_ _] true)
-
-(defmethod driver/database-supports? [:firebolt :foreign-keys]  [_ _] true)
-
-(defmethod driver/database-supports? [:firebolt :binning]  [_ _] true)
+(defmethod driver/database-supports? [:firebolt :basic-aggregations]  [_ _ _] true)
 
-(defmethod driver/database-supports? [:firebolt :regex]  [_ _] true)
+(defmethod driver/database-supports? [:firebolt :expression-aggregations]  [_ _ _] true)
 
-(defmethod driver/database-supports? [:firebolt :standard-deviation-aggregations]  [_ _] false)
+(defmethod driver/database-supports? [:firebolt :percentile-aggregations]  [_ _ _] true)
 
-(defmethod driver/database-supports? [:firebolt :nested-queries]  [_ _] false)
+(defmethod driver/database-supports? [:firebolt :foreign-keys]  [_ _ _] true)
 
-(defmethod driver/database-supports? [:firebolt :case-sensitivity-string-filter-options]  [_ _] false)
+(defmethod driver/database-supports? [:firebolt :binning]  [_ _ _] true)
 
-(defmethod driver/database-supports? [:firebolt :set-timezone]  [_ _] false)
+(defmethod driver/database-supports? [:firebolt :regex]  [_ _ _] true)
 
-(defmethod driver/database-supports? [:firebolt :nested-fields]  [_ _] false)
+(defmethod driver/database-supports? [:firebolt :standard-deviation-aggregations]  [_ _ _] false)
 
+(defmethod driver/database-supports? [:firebolt :nested-queries]  [_ _ _] false)
 
+(defmethod driver/database-supports? [:firebolt :case-sensitivity-string-filter-options]  [_ _ _] false)
 
+(defmethod driver/database-supports? [:firebolt :set-timezone]  [_ _ _] false)
```

Comment:
I think this is supported, right?

Reply:
This needs some additional implementation to make it enabled. We can do this as a part of a separate task

Reply:
https://packboard.atlassian.net/browse/FIR-35123